docs/Getting_Started.md

```diff
@@ -0,0 +1,41 @@
+# Getting Started
+ 
+ ## Microcontrollers
+ KMK will run on most microcontrollers supported by [Circuitpython](https://circuitpython.org/downloads). Our recommended microcontrollers are found [here](Officially_Supported_Microcontrollers.md)
+ 
+ ## Firmware
+ ### Circuitpython
+Circuitpython is what KMK will run on top of. Make sure that Circuitpython is installed on your 
+device using the guide [here](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython). 
+It's recommended to run the latest stable version that is at least 5.0 or higher. Beta versions may work, but expect limited support.
+ 
+ ### KMKPython
+KMKPython is a fork of Circuitpython, but with libraries for most extensions built in. This saves you from having to get them all 
+and keep them updated yourself. There may be other features added in the future that are exclusive to KMKPython.
+ 
+ ## Getting KMK
+ You can always find the latest releases on our CDN, in [compiled and optimized](https://cdn.kmkfw.io/kmk-latest.zip) and 
+ [raw, hackable text file](https://cdn.kmkfw.io/kmk-latest.unoptimized.zip) forms. These follow the `master` branch here on GitHub. 
+ Just get the KMK folder and drop this directly in the CIRCUITPYTHON directory (not in a sub folder). Make sure to extract the zip, 
+ and put the `kmk` folder on the root of the CIRCUITPY drive on the microcontroller
+
+## Turning a controller into a keyboard
+### Supported keyboards
+If your keyboard and microcontroller are officially supported, it's as easy as visiting the page for your files, and dropping them 
+on the root of the "flash drive". Those pages can be found [here](https://github.com/KMKfw/boards). You will need the `kb.py` and `main.py`. More advanced instructions
```

Comment:
This is to allow for boards with multiple controllers. I can have a single main.py with a seperate kb.py for nice!nano, and another for itsybitsy as there will be some boards that can take both, but act the same otherwise. This also allows converter/nicenano with a single keymap.

Reply:
There are now examples under /board such as the keyboard that you use. both converter board itsy and nice!nano are supported and the same main.py is reused depending on the file picked.